api/default_test.go

```diff
@@ -11,14 +11,15 @@ import (
 
 func TestOverrideApiConfig(t *testing.T) {
 	newNodeConfig := &params.NodeConfig{
-		APIModules:      "test, eth, wakuv2",
-		ConnectorConfig: params.ConnectorConfig{Enabled: true},
-		HTTPEnabled:     true,
-		HTTPHost:        "0.0.0.0",
-		HTTPPort:        8545,
-		WSEnabled:       false,
-		WSHost:          "127.0.0.1",
-		WSPort:          8586,
+		APIModules:       "test, eth, wakuv2",
+		ConnectorConfig:  params.ConnectorConfig{Enabled: true},
+		HTTPEnabled:      true,
+		HTTPHost:         "0.0.0.0",
+		HTTPPort:         8545,
+		HTTPVirtualHosts: []string{"status-go"},
```

Comment:
Is `"status-go"` the default value? Not an empty list?

Reply:
Honestly, we should just use `"*"` for tests and not think about it.